examples/arrays/arrays.md

```diff
@@ -72,6 +72,15 @@ println('Thiago' in names)
 >> true
 ```
 
+## Create an empty array
```

Comment:
I think this piece could probably be to the `How to declare an array` section.

Reply:
ya.. great idea !!! :+1:

Reply:
im using header level 3

Reply:
Agreed with @vbrazo